docs/current/sdk/elixir/756758-get-started.md

```diff
@@ -0,0 +1,117 @@
+
+- Install the Elixir SDK
+- Create an Elixir CI tool to test an application
+- Improve the Elixir CI tool to test the application against multiple Elixir and OTP versions
+
+## Requirements
+
+This tutorial assumes that:
+
+- You have a basic understanding of the Elixir programming language. If not, [read the Elixir documentation](https://elixir-lang.org/learning.html).
+- You have an Elixir development environment with Elixir 1.14 or later and Erlang/OTP 25 or later. If not, install [Elixir](https://elixir-lang.org/install.html) and [Erlang/OTP](https://www.erlang.org/downloads).
+- You have the Dagger CLI installed on the host system. If not, [install the Dagger CLI](../../cli/465058-install.md).
+- You have Docker installed and running on the host system. If not, [install Docker](https://docs.docker.com/engine/install/).
+
+## Step 1: Create an Elixir project
+
+Create an Elixir project with `mix`:
+
+```shell
+mix new a_project
+```
+
+## Step 2: Install the Dagger Elixir SDK
+
+In your project directory, open `mix.exs` and add `{:dagger, "~> 0.8"}` to the list in the `deps` function:
+
+```elixir
+def deps do
+  [
+    {:dagger, "~> 0.8", only: [:dev, :test]}
+  ]
+end
+```
+
+Run `mix deps.get` to fetch the Elixir SDK from Hex.pm:
+
+```shell
+mix deps.get
+```
+
+## Step 3: Create the Mix task
+
+Create a new module and Mix task to test the project at `lib/mix/tasks/a_project.test.ex`:
+
+```elixir file=snippets/get-started/step3/a_project.test.ex
+```
+
+This module performs the following operations:
+
+- It starts all applications related to the Dagger Elixir SDK with `Application.ensure_all_started(:dagger)`.
+- It creates a Dagger client with `Dagger.connect!/1`.
+- It uses the client's `Dagger.Client.host/1` and `Dagger.Host.directory/3` to obtain a reference to the host directory. It additionally uses the `exclude` filter in `Dagger.Host.directory/3` to filter unwanted files and directories.
+- It uses the client's `Dagger.Client.container/2` and `Dagger.Container.from/2` to initialize a new container from the `hexpm/elixir:1.15.4-erlang-25.3.2.5-ubuntu-bionic-20230126` base image.
+- It uses `Dagger.Container.with_mounted_directory/3` to mount the project's source files into the container.
+- It uses `Dagger.Container.with_exec/3` to define the commands to be executed in the container - in this case, commands such as `mix deps get`, which downloads dependencies and `mix test`, which runs unit tests. Each invocation of `with_exec` returns a revised `Container` with the results of command execution.
+- It uses `Dagger.Sync.sync/1` to force command execution.
+- It uses `Dagger.close/1` to close the client connection.
+
+Run the Mix task by executing the command below from the project directory:
+
+```shell
+dagger run mix a_project.test
+```
+
+The `dagger run` command executes the specified command in a Dagger session and displays live progress. Here is an example of the output:
+
+```shell
+Tests succeeded!
```

Comment:
@wingyplus following a structure similar to that the Python SDK get started, can you please update your code to display this output at the end?